cli/src/test/resources/ide-projects/jasypt/_ide/urls/jasypt/jasypt/1.9.3/status.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
none of these files in `_ide/urls` is actually used by your tests. Hence, you should remove them to avoid pointless clutter.